docs/usage-wharfyml/step-types/container.md

```diff
@@ -10,5 +10,8 @@ container:
   # Optional arguments
   os: linux/windows # (defaults to linux)
   shell: /bin/sh
+  secretName: # if specified, the target Kubernetes secret name will be mounted inside the container as environment variables. The secret must have the name 'wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-${secretName}'.
```

Comment:
```suggestion
  secretName: # if specified, the target Kubernetes secret name will be mounted inside the container as environment variables. The secret must have the name 'wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-${someSecretName}'.
```

Maybe this is clearer, otherwise it might look like you reference the parameter named secretName.

Reply:
Ooo haven't seen GitHub suggestions in action yet.

Using the variable syntax and referencing some undefined variable seems confusing to me. I tried to reference the property name here, as I've done in some other descriptions as well in the wharf-ci-schema.json.

Maybe if I add "where `${secretName}` is the value of this setting"?

Reply:
Otherwise just `wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-mysecret` and let users understand themselves that they can use variables if they want? Otherwise it might be perceived as you **have** to use a variable for that last part.

Reply:
Oooh never mind, I thought that you had to do this:

```yml
secretName: wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-somesecret
```

But what you should do is just

```yml
secretName: somesecret
```

Right?

Then you could just disregard this feedback, unless you think you can avoid people getting confused like me (but it could just be me being stupid)

Maybe just this would help:

```suggestion
  secretName: mysecret # if specified, the target Kubernetes secret name will be mounted inside the container as environment variables. The secret must have the name 'wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-mysecret'.
```

Reply:
That last one was an excellent suggestion. I applied it